scripts/deploy.js

```diff
@@ -0,0 +1,46 @@
+const hre = require("hardhat");
+const { ethers } = hre;
+
+const deployConnector = require("./deployConnector");
+
+async function main() {
+    const accounts = await hre.ethers.getSigners()
+    const wallet = accounts[0]
+    
+    const connectMapping = {
+        '1inch': 'ConnectV2OneInch',
+        'aaveV1': 'ConnectV2AaveV1',
+        'aaveV2': 'ConnectV2AaveV2',
+        'auth': 'ConnectV2Auth',
+        'basic': 'ConnectV2Basic',
+        'comp': 'ConnectV2COMP',
+        'compound': 'ConnectV2Compound',
+        'dydx': 'ConnectV2Dydx',
+        'fee': 'ConnectV2Fee',
+        'gelato': 'ConnectV2Gelato',
+        'maker': 'ConnectV2Maker',
+        'uniswap': 'ConnectV2UniswapV2'
```

Comment:
```suggestion
        '1inch-A': 'ConnectV2OneInch',
        'aave-v1-A': 'ConnectV2AaveV1',
        'aave-v2-A': 'ConnectV2AaveV2',
        'authority-A': 'ConnectV2Auth',
        'basic-A': 'ConnectV2Basic',
        'comp-A': 'ConnectV2COMP',
        'compound-A': 'ConnectV2Compound',
        'dydx-A': 'ConnectV2Dydx',
        'fee-A': 'ConnectV2Fee',
        'gelato-A': 'ConnectV2Gelato',
        'makerdao-A': 'ConnectV2Maker',
        'uniswap-A': 'ConnectV2UniswapV2'
```

Reply:
@KaymasJain Once can you verify the connector names?

Reply:
just a suggestion (not sure if make sense) - what if we keep everything uppercase, does it look better?

AAVE-V1-A

Reply:
Okay! Let's go with the Sowmay one @thrilok209?

Reply:
@Sowmayjain @KaymasJain so all in upper case, right?

Reply:
If you agree, yes.

Reply:
Alright. We can do that. Will make the changes.